docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - SANDBOX_MODE=${SANDBOX_MODE}
       - BRAPI_REFERENCE_SOURCE=${BRAPI_REFERENCE_SOURCE}
       - BRAPI_VENDOR_SUBMISSION_ENABLED=${BRAPI_VENDOR_SUBMISSION_ENABLED:-false}
+      - VUE_APP_ALTERNATE_AUTHENTICATION_ENABLED=${VUE_APP_ALTERNATE_AUTHENTICATION_ENABLED:-false}
```

Comment:
I think this should just be `ALTERNATE_AUTHENTICATION_ENABLED` without the  `VUE_APP_` prefix like the others.